### ISSUE TITLE
Memory Management BSOD after loading model with exllamav2

### ISSUE THREAD

**Issue by Norwaere** [oobabooga/text-generation-webui#5501]
### Describe the bug

My PC crashes after loading any GPTQ format model that requires multiple gpus worth of vram with exllamav2. This problem appeared after updating to any version of exllamav2 newer than version 0.0.11. Exl2 models work like they used to.
I observed this behaviour with the models:
TheBloke_Mixtral-8x7B-Instruct-v0.1-GPTQ_gptq-4bit-32g-actorder_True
TheBloke_Mixtral_34Bx2_MoE_60B-GPTQ_gptq-3bit-128g-actorder_True

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Load gptq model that is larger than vram of the main gpu (12gb)

### Logs

```shell
none available. Windows even fails to provide a crashlog after this particular bsod.
```

### System Info

```shell
Nvidia RTX 4070, Nvidia RTX 3050, Nvidia Tesla P100, GPU driver 537.13, AMD Ryzen 5600X, 32GB Ram, Windows 10 22H2
```

**Comment by Annamae-beep** [comment]
I'm experiencing a similar issue with 2x Nvidia 4060's (32GB in total), 64GB Ram.
I can run a smaller model in either GPU without issues, but using both for a larger model causes BSOD.
Everything worked fine before, until I decided to Update Ooba.

I see that this issue has been marked as a bug - here's hoping they'll be a fix out soon, coz I'm pulling my hair out trying to resolve it my end.

**Comment by Norwaere** [comment]
downgrading to Exllamav2 0.0.11 worked or just using a model with the exl2 format on the most recent version of exllamav2 works seems to avoid this issue

**Comment by Annamae-beep** [comment]
Yes I agree and thank you, downgrading is probably the best option but I have no idea how to integrate an older version of exllamav2 into Ooba.
The current version works fine on a single GPU, but seems unable to manage memory across multiple GPU's, at least on some systems including mine.

**Comment by 2Barry2** [comment]
Still relevant. version 0.0.20, Mem Management BSOD on Windows when trying to split a 70B exl2 model into two 3090 GPUs (split: 22, 22)

**Comment by Goldenkoron** [comment]
Still relevant. Memory management bluescreens about 25% of the time when loading a exllamav2 model, always happens when third gpu in the device order starts loading. Changing the device order doesn't change it though, it always happens on the last gpu to start loading.
